src/socket.h

```diff
@@ -31,6 +31,9 @@ typedef struct _pthreads_socket_t {
 	zend_bool blocking;
 } pthreads_socket_t;
 
+#define PHP_NORMAL_READ 0x0001
+#define PHP_BINARY_READ 0x0002
```

Comment:
Given that these exact constants are already defined in ext/socket, it may be best to rename them to something else (it would only potentially pose a problematic when both pthreads and ext/sockets are statically compiled with PHP).